internal/bminventory/inventory.go

```diff
@@ -2035,6 +2036,10 @@ func (b *bareMetalInventory) updateNetworkParams(params installer.UpdateClusterP
 		machineCidr = ""
 		setMachineNetworkCIDRForUpdate(updates, machineCidr)
 	}
+	if params.ClusterUpdateParams.NetworkType != nil && params.ClusterUpdateParams.NetworkType != cluster.NetworkType {
+		updates["network_type"] = swag.StringValue(params.ClusterUpdateParams.NetworkType)
+	}
+
```

Comment:
Does this feature also need to be reported to feature_usage?

Reply:
good point - I think we should add it

Reply:
I see a separate ticket was added.

Reply:
better to add it here . it's a one liner and has to do with the feature itself